target/linux/ramips/image/rt305x.mk

```diff
@@ -1086,6 +1086,20 @@ define Device/unbranded_a5-v11
 endef
 TARGET_DEVICES += unbranded_a5-v11
 
+define Device/tenda_a6
+  SOC := rt5350
+  IMAGE_SIZE := 3776k
+  IMAGES += factory.bin
+  IMAGE/factory.bin := $$(sysupgrade_bin) | check-size | \
+	poray-header -B A5-V11 -F 4M
+  DEVICE_VENDOR := Tenda
+  DEVICE_MODEL := A6
+  DEVICE_PACKAGES := kmod-usb-ohci kmod-usb2
+  SUPPORTED_DEVICES += a6
```

Comment:
This line can be dropped.

Reply:
Which one? You quoted 4. The device has an unpopulated USB header, but the SoC has working USB support, so it's probably best to include the modules

Reply:
The line directly before the comment.

SUPPORTED_DEVICES in set automatically based on the device node name. Despite, the value you provide does not match anything anyway.